pkg/metrics/schema.go

```diff
@@ -3,38 +3,40 @@ package metrics
 const (
 	// METRIC NAMES.
 
-	// SignalReadingMetricName - used in circuit metrics.
-	SignalReadingMetricName = "signal_reading"
+	// HTTP metrics.
+
+	// HTTPRequestMetricName - metric from http server.
+	HTTPRequestMetricName = "http_requests_total"
+	// HTTPRequestLatencyMetricName - metric from http server.
+	HTTPRequestLatencyMetricName = "http_requests_latency_seconds"
```

Comment:
This is ms as well. Always refer [where its used](https://github.com/fluxninja/aperture/blob/828775e24d588c26e7a967bbbe9adb4676c8a203/pkg/net/http/server.go#L224) to double check